### PR TITLE
Add Day03 by Samuel Chassot

### PR DESCRIPTION
**Pull request by samuelchassot**

### CODE REVIEW

**Review by gkepka — CHANGES_REQUESTED** [review]

File: docs/2024/puzzles/day03.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ https://adventofcode.com/2024/day/3
 - [Solution](https://github.com/makingthematrix/AdventOfCode2024/blob/main/src/main/scala/io/github/makingthematrix/AdventofCode2024/DayThree.scala) by [Maciej Gorywoda](https://github.com/makingthematrix)
 - [Solution](https://github.com/YannMoisan/advent-of-code/blob/master/2024/src/main/scala/Day3.scala) by [YannMoisan](https://github.com/YannMoisan)
 - [Solution](https://github.com/spamegg1/aoc/blob/master/2024/03/03.worksheet.sc#L62) by [Spamegg](https://github.com/spamegg1/)
+- [Solution](https://github.com/samuelchassot/AdventCode_2024/blob/c213b5ca3cf7eabd3c2d2ba852a36869575b1442/03/Day03.scala) by [Samuel Chassot](https://github.com/samuelchassot/)
```
[the remark this answers is not in the document]
Comment (samuelchassot) [reply docs/2024/puzzles/day03.md:18]: that's a github permalink. Is it an issue?

[the remark this answers is not in the document]
Comment (samuelchassot) [reply docs/2024/puzzles/day03.md:18]: ahh but my repo is private, sorry about that. Let me change the permissions!

[the remark this answers is not in the document]
Comment (samuelchassot) [reply docs/2024/puzzles/day03.md:18]: now it should work! Sorry again!

[the remark this answers is not in the document]
Comment (gkepka) [reply docs/2024/puzzles/day03.md:18]: Yep, everything is fine now

**Review by gkepka — APPROVED** [review]

**Pull request merged**